pkg/storaged/overview.js

```diff
@@ -417,6 +418,21 @@ define([
                                 }
                               }
                           ],
+                          ChangeHandler: function (dialog) {
+                              // hide the chunk size selector for RAID 1, as the chunk size setting doesn't make sense
```

Comment:
I was thinking of this: https://github.com/cockpit-project/cockpit/compare/master...mvollmer:storage-no-chunk-for-raid1